pigeons/pigeons.dart

```diff
@@ -141,8 +151,9 @@ class WatchResource {
 class InstallData {
   String uri;
   PbwAppInfo appInfo;
+  String? appstoreId;
```

Comment:
Why don't you use AppstoreData pigeon object instead of manually serializing to/from json here?